frontend/src/components/SideNav.tsx

```diff
@@ -548,6 +548,18 @@ export default class SideNav extends React.Component<SideNavProps, SideNavState>
               </div>
             </Tooltip>
           )}
+          <Tooltip
+              title="Report an Issue"
+              enterDelay={300}
+              placement={'top-start'}>
+            <div className={css.envMetadata}>
+              <a href="https://github.com/kubeflow/pipelines/issues/new?template=BUG_REPORT.md"
```

Comment:
Please add `rel="noopener"` in the `<a>`. It helps fix a security risk.

Ref: https://mathiasbynens.github.io/rel-noopener/

Reply:
probably also, the above two links